README.md

```diff
@@ -16,59 +16,59 @@ Note: If you want to use the plugin on macOS, see our [advice](#macos-advice).
 *Alternatively, copy the plugin to {YourGameRoot}/Plugins.*
 3. Launch your game project. If you are starting out from scratch, create a game using the First Person template.
 4. If the plugin is not enabled automatically, enable it manually in the Edit->Plugins window and restart Unreal Editor.
-5. Create a new Blueprint class and choose DolbyIO as parent class.
+5. Create a new Blueprint class and choose DolbyIO as the parent class.
 6. In the Event Graph of the newly created Blueprint, hover over Functions on the left and select Override->[On Token Needed](#on-token-needed). You will also need to override [On Initialized](#on-initialized). Both events should appear in the Blueprint. There are more [events](#events), but to get started, you only need to handle these two.
 
 ## Events
 ### On Token Needed
-An initial or refreshed [client access token](https://docs.dolby.io/communications-apis/docs/overview-developer-tools#client-access-token) is needed. Triggered when the game starts or when a refresh token is requested.
+Triggered when an initial or refreshed [client access token](https://docs.dolby.io/communications-apis/docs/overview-developer-tools#client-access-token) is needed, which happens when the game starts or when a refresh token is requested.
 
-When this event is received, you should obtain a token for your Dolby.io application and call the [Set Token](#set-token) function.
+After receiving this event, obtain a token for your Dolby.io application and call the [Set Token](#set-token) function.
 
 Example:  
 ![example](Images/on_token_needed.PNG)
 
 ### On Initialized
-The plugin is successfully initialized. Triggered by the [Set Token](#set-token) function.
+Triggered when the plugin is successfully initialized after calling the [Set Token](#set-token) function.
 
-When this event is received, the plugin is ready for use. You can now, for example, call this Blueprint's [Connect](#connect) function. Once connected, the [On Connected](#on-connected) event will trigger.
+After receiving this event, the plugin is ready for use. You can now, for example, call this Blueprint's [Connect](#connect) function. Once connected, the [On Connected](#on-connected) event will trigger.
 
 Example:  
 ![example](Images/on_initialized.PNG)
 
 ### On Connected
-Successfully connected to conference. Triggered by the [Connect](#connect) function. Provides the ID of the local participant in its argument.
+Triggered when a client is successfully connected to a conference after calling the [Connect](#connect) function. The event provides the ID of the local participant in its argument.
```

Comment:
"A client successfully connected to a conference" sounds like it applies to any client in any ongoing conference. There can only be at most one client and one conference in the plugin at any given moment, so I think "the" is correct article in both cases. The same applies to the two following events.